tools/registry.py

```diff
@@ -264,20 +268,35 @@ def add(self, module, override=False):
     if module.presubmit_yml:
       shutil.copy(module.presubmit_yml, presubmit_yml)
     else:
-      platforms = {
-          "centos7": {},
-          "debian10": {},
-          "ubuntu2004": {},
-          "macos": {},
-          "windows": {},
-      }
-      for _, config in platforms.items():
-        if module.build_targets:
-          config["build_targets"] = module.build_targets.copy()
-        if module.test_targets:
-          config["test_targets"] = module.test_targets.copy()
+      PLATFORMS = ["centos7", "debian10", "ubuntu2004", "macos", "windows"]
+      presubmit = dict()
```

Comment:
sorry, I didn't mean that you should replace `{}` literals with `dict()`; what I meant was that instead of:

```python
presubmit = {}
presubmit["matrix"] = {}
presubmit["matrix"]["platform"] = PLATFORMS.copy()
presubmit["tasks"] = {}
```

we could write:

```python
presubmit = {
  "matrix": {
    "platform": PLATFORMS.copy(),
  },
  "tasks": {},
}
```

but it's not a big deal.

Reply:
😅 Sorry for the misunderstanding..